playbooks/common/openshift-cluster/upgrades/post_control_plane.yml

```diff
@@ -51,6 +51,9 @@
       '{"spec":{"template":{"spec":{"containers":[{"name":"router","image":"{{ router_image }}","livenessProbe":{"tcpSocket":null,"httpGet":{"path": "/healthz", "port": 1936, "host": "localhost", "scheme": "HTTP"},"initialDelaySeconds":10,"timeoutSeconds":1}}]}}}}'
       --api-version=v1
     with_items: "{{ haproxy_routers }}"
+    # AUDIT:changed_when_note: `false` not being set here. What we
```

Comment:
Yes Scott. I believe it was during the friday call that we agreed we would do a future sweet for all `changed_when` statements because we have tonnnnnnnnnnnnnns of 'em.